test/test_site/expected/testTags.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta name="default-head-top">
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Hello World</title>
+    <link rel="stylesheet" href="markbind\css\bootstrap.min.css">
+    <link rel="stylesheet" href="markbind\css\bootstrap-vue.min.css">
+    <link rel="stylesheet" href="markbind\css\font-awesome.min.css" >
+    <link rel="stylesheet" href="markbind\css\bootstrap-glyphicons.min.css" >
+    <link rel="stylesheet" href="markbind\css\github.min.css">
+    <link rel="stylesheet" href="markbind\css\markbind.css">
+    <link rel="stylesheet" href="markbind\layouts\default\styles.css">
+    
+    <meta name="default-head-bottom">
+    <link rel="icon" href="/test_site/favicon.png">
+</head>
+<body>
+<div id="app">
+    <div id="content-wrapper">
+  <h1 id="front-matter-should-be-overridden-by-site.json">Front matter should be overridden by site.json<a class="fa fa-anchor" href="#front-matter-should-be-overridden-by-site.json"></a></h1>
+  <div>
+    <h1 id="div-with-shown-tag">Div with shown tag<a class="fa fa-anchor" href="#div-with-shown-tag"></a></h1>
+    <div tags="tag--shown">
+      Div with shown tag
+    </div>
+    <h1 id="div-with-shown-2-tag">Div with shown-2 tag<a class="fa fa-anchor" href="#div-with-shown-2-tag"></a></h1>
+    <div tags="tag--shown-2">
+      Div with shown-2 tag
+    </div>
+    <h1 id="div-with-multiple-tags">Div with multiple tags<a class="fa fa-anchor" href="#div-with-multiple-tags"></a></h1>
+    <div tags="tag--shown tag--other">
+      Div with multiple tags
+    </div>
+    <h1 id="div-with-should-be-overridden-tag">Div with should be overridden tag<a class="fa fa-anchor" href="#div-with-should-be-overridden-tag"></a></h1>
+
+    <h1 id="div-with-hidden-tag">Div with hidden tag<a class="fa fa-anchor" href="#div-with-hidden-tag"></a></h1>
```

Comment:
- Why are `div-with-should-be-overridden-tag` and `div-with-hidden-tag` rendered?
- What's overridden?

Reply:
Ah, thanks for the clarification!